### PR TITLE
Add Participants feature to Assignment and Courses page

### PR DESCRIPTION
**Pull request by AtharvaThorve**
More details about issue #14 

[Documentation](https://wiki-expertiza.csc.ncsu.edu/index.php/CSC/ECE_517_Fall_2023_-_E2364._Create_a_UI_for_Course%27s_%26_Assignment%27s_"Add_Participants"_page)

Mentor: Renji Joseph Sabu

Team Name: Codechamps
Team Members:
- Atharva Jaysing Thorve
- Mrityunjay Joshi
- Divit Kalathil

### CODE REVIEW

File: src/pages/Participants/participantUtil.ts [hunk 1]
```diff
@@ -0,0 +1,124 @@
+import { IFormOption } from "components/Form/interfaces";
+import axiosClient from "../../utils/axios_client";
+import { IInstitution, IRole, IParticipantRequest, IParticipantResponse } from "../../utils/interfaces";
+
+/**
+ * @author Ankur Mundra on April, 2023
```
Comment (renjiniravath) [inline src/pages/Participants/participantUtil.ts]: This is a new file, right?

Comment (AtharvaThorve) [reply src/pages/Participants/participantUtil.ts]: Yes
We should change the author name right?

Comment (renjiniravath) [reply src/pages/Participants/participantUtil.ts]: Yup

Comment (AtharvaThorve) [reply src/pages/Participants/participantUtil.ts]: Fixed it.

File: src/App.tsx [hunk 2]
```diff
@@ -1,22 +1,24 @@
-import React from "react";
-import Home from "pages/Home";
```
Comment (renjiniravath) [inline src/App.tsx]: If possible, can you avoid these unwanted changes due to change in order of imports? It affects the readability of the pull request.

Comment (AtharvaThorve) [reply src/App.tsx]: Sorry for the delay. It has been fixed.

**Review by kalyan-karnati — APPROVED** [review]

**Pull request merged**